components/shooter.py

```diff
@@ -0,0 +1,32 @@
+from magicbot import tunable
+import wpilib
+
+
+class Shooter:
+    shooter_motor: wpilib.Talon
+    shoot_speed = tunable(1.0)  # speed is tunable via NetworkTables
+
+    def __init__(self):
+        self.deployed = False
+
+    def shoot(self):
+        pass
+
+    def stop(self):
+        pass
+
+    def deploy(self):
+        """Causes the shooter motor to spin"""
+        self.deployed = True
+
+    def is_ready(self):
```

Comment:
I think we should make these so-called "feedback" functions with feedback decorators from magicbot

e.g.
`@feedback`
`def is_ready(self) -> bool`